cartridges/cloudinary_sg_changes/cartridge/templates/default/product/components/productimages.isml

```diff
@@ -51,13 +51,11 @@
     <iscomment>Custom Start: Cloudinary Integration</iscomment>
     <a ${href} class="product-image main-image ${pdict.cloudinary && pdict.cloudinary.isEnabled ? ('main-image-' + pdict.Product.ID) : ''}" title="${imageTitle}">
         <img
-            itemprop="image"
             class="primary-image ${pdict.cloudinary && pdict.cloudinary.isEnabled ? ('cld-responsive primary-image-' + pdict.Product.ID) : ''}"
-            <isif condition="${pdict.cloudinary && pdict.cloudinary.isEnabled}">
-                data-src="${imageUrl}"
-            </isif>
+            data-src="${imageUrl}"
             src="${imageUrl}"
```

Comment:
where is the 'srcset' and 'sizes' attributes?

Reply:
It's not there, from the start. If you really need we can add that.

Reply:
yes, it's mandatory